test/docfx.Test/TestUtility.cs

```diff
@@ -188,6 +189,20 @@ public static JToken ApplyVariables(JToken value, IEnumerable<KeyValuePair<strin
             return value;
         }
 
+        public static string RandomSHA1Hash()
+        {
```

Comment:
don't use SHA1 even for non-crypto scenarios - this will raise red flag in security reportings

Reply:
lemme figure out another way to generate a 40 digits hex string

Reply:
using random 40 digits hex string